mod/module_test.go

```diff
@@ -147,3 +150,22 @@ func TestHasPathPrefix(t *testing.T) {
 
 	assert.False(t, hasPathPrefix("github.com/anz-bank/sysl2", "github.com/anz-bank/sysl/deps.sysl"))
 }
+
+func removeFile(t *testing.T, fs afero.Fs, file string, isDir bool) {
+	if isDir {
+		exists, err := afero.DirExists(fs, file)
+		assert.NoError(t, err)
```

Comment:
this should use `require.NoError` to make sure that the program stops if there's an error

Reply:
any `assert.NoError` in this function should get the same changes